sql-plugin/src/main/format/ShuffleMetadataResponse.fbs

```diff
@@ -16,46 +16,16 @@ include "ShuffleCommon.fbs";
 
 namespace com.nvidia.spark.rapids.format;
 
-/// Metadata about cuDF Columns
-table ColumnMeta {
-  /// number of nulls in the column or -1 if unknown
-  null_count: long;
-
-  /// number of rows in the column
-  row_count: long;
-
-  /// offset of the column's data buffer
-  data_offset: long;
-
-  /// length of the column's data buffer
-  data_length: long;
-
-  /// offset of the column's validity buffer
-  validity_offset: long;
-
-  /// offset of the column's offsets buffer
-  offsets_offset: long;
-
-  /// child column metadata
-  children: [ColumnMeta];
-
-  /// ordinal of DType enum
-  dtype_id: int;
-
-  /// DType scale for decimal types
-  dtype_scale: int;
-}
-
 /// Metadata about cuDF tables
 table TableMeta {
-  /// metadata about the table buffer
+  /// metadata about the data encoding
   buffer_meta: BufferMeta;
 
-  /// metadata for each column in the table buffer
-  column_metas: [ColumnMeta];
-
   /// number of rows in the table
   row_count: long;
+
+  /// opaque metadata describing the packed table schema and data layout
+  packed_meta: [byte];
```

Comment:
I haven't explicitly examined the payload size.  Each column is encoded as a C++ struct in the metadata, and [eyeballing the code](https://github.com/rapidsai/cudf/pull/7096/files#diff-e7e0c690336137aefd4c8099055fb51b39f5f32106034287f651b5fb8442412cR27-R61) I'm guessing it's around 40 bytes per column, nested or otherwise, in the table.